api/types.proto

```diff
@@ -514,6 +514,15 @@ message TaskStatus {
 	// HostPorts provides a list of ports allocated at the host
 	// level.
 	PortStatus port_status = 6;
+
+	// AppliedBy gives the node ID of the manager that applied this task
+	// status update to the Task object.
+	string applied_by = 7;
```

Comment:
That'd be a cool thing to have eventually - would we store historical skew data between the manager nodes for the adjustment?